mitxpro/settings.py

```diff
@@ -260,6 +260,10 @@
     0,
     description="Maximum age of connection to Postgres in seconds",
 )
+DEFAULT_DATABASE_CONFIG["DISABLE_SERVER_SIDE_CURSORS"] = get_bool(
+    "MITXPRO_DB_DISABLE_SS_CURSORS", True
```

Comment:
Can you add a description here? Something like `If True, disables server-side database cursors to prevent invalid cursor errors when using pgbouncer`